dev/tests/unit/Magento/FunctionalTestFramework/StaticCheck/AnnotationsCheckTest.php

```diff
@@ -56,11 +51,9 @@ public function testValidateRequiredAnnotationsNoError()
         ];
         $expected = [];
 
-        // mock test object
-        $test = AspectMock::double(
-            TestObject::class,
-            ['getAnnotations' => $annotations, 'getName' => 'AnnotationsCheckTest']
-        )->make();
+        $test = $this->createMock(TestObject::class);
+
+        $test->expects($this->once())->method('getAnnotations')->willReturn($annotations);
```

Comment:
Can we also add `$test->expects($this->once())->method('getName')->willReturn('AnnotationsCheckTest');`?

Reply:
Never mind. Actually getName() is not invoked in this test case.

Reply:
Yes, I added that expectation with a `once` and the test failed, that's why I removed it. :)